MoPubSDK/NativeVideo/Internal/MOPUBPlayerViewController.h

```diff
@@ -31,6 +31,17 @@
 
 @end
 
+@protocol MOPUBPlayerViewControllerExposerDelegate <NSObject>
+
+@optional
+
+- (void)playerPlaybackDidStart:(MOPUBPlayerViewController *)player;
+- (void)playerDidProgressToTime:(NSTimeInterval)playbackTime;
```

Comment:
By convention, I suggest including a reference to the player itself, i.e., the signature should be `player:didProgressToTime:`.